documentation/manual/compile-with-cmake.rst

```diff
@@ -181,6 +178,61 @@ is a translation guide between the frontend script and "bare" CMake:
 - ``--cmake-options="-DTHIS -DTHAT"``. Sets options to be forwarded as-is to
   CMake.
 
+Installation
+------------
+
+It is possible to install HANDE either running outside the ``build`` directory:
+
+.. code-block:: bash
+
+   $ cmake --build build --target install
+
+or from within the ``build`` directory:
+
+.. code-block:: bash
+
+   $ make install
+
+By default, CMake will set the install prefix to ``/usr/local`` and you might
+hence not have permissions to successfully install. The install prefix can be
+set *via* the ``--prefix`` option to the frontend script or, equivalently,
+passing the desired path to CMake *via* the ``CMAKE_INSTALL_PREFIX`` variable.
+The HANDE executable and static library will be installed to the ``bin`` and
+``lib`` (``lib64`` on 64-bit systems) subdirectories of the install prefix.
+
+.. note::
+
+    It might be advisable to install pyhande to the same prefix as the HANDE
+    executable.
```

Comment:
Probably outside scope here but it would be nice to link the pyhande docs here and have them include the install info (which is currently just in the README)